microsvcs/admin/forum/app/routes.js

```diff
@@ -15,7 +15,7 @@ module.exports = app => {
     router.delete("/posts/:post_id/:uid", FORUM.deletePost);
 
     // Comment Specific
-    router.put("/posts/:post_id/comments/:uid", COMMENTS.addComment);
```

Comment:
Can consider POST `/posts/:post_id/comments`

Reply:
Any reason why you choose to use put instead post, since it is adding comments but not editing them?

Reply:
ah this one was related to the behaviour for inserting a new comment.

since currently its an array of objects within the original document, i took it as updating the original document, hence the put, but I think that post also makes sense in hindsight, since its also adding a new entry? (will switch over to post if the separate collection is implemented. i think that would make more sense for the behaviour)

will look into restructuring the comments 🤔

Reply:
updated to post